bundle/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -1293,13 +1293,7 @@ spec:
                 - mountPath: /var/run/secrets/serving-cert
                   name: serving-cert
                   readOnly: true
-              nodeSelector:
-                node-role.kubernetes.io/master: ""
```

Comment:
Sorry I think I missed this, is this an expected behavior?

Reply:
I wasn't expecting this to change. Was this due to the patch for adding hypershift deployment support?

Reply:
I tried to fix this with https://github.com/ComplianceAsCode/compliance-operator/pull/346 btw if we merged https://github.com/ComplianceAsCode/compliance-operator/pull/303 we would probably be alerted of changes like these sooner.

Reply:
Even though #303  is still in flight, I'm going to rebase this change so we can pickup https://github.com/ComplianceAsCode/compliance-operator/pull/346